### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by CanerKaraca23**

### CODE REVIEW

File: .github/workflows/beta.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   build:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```
Comment (authorisation) [inline .github/workflows/beta.yml:12]: Change this back.

**Review by authorisation — CHANGES_REQUESTED** [review]

**Comment by authorisation** [comment]
Have you tested this?

**Comment by CanerKaraca23** [comment]
> Have you tested this?

Yes.

**Review by authorisation — APPROVED** [review]

**Review by authorisation — CHANGES_REQUESTED** [review]
Explanation required on why certain dependencies were downgraded

**Comment by CanerKaraca23** [comment]
> Explanation required on why certain dependencies were downgraded

Which dependencies are you talking about? None of them downgraded, you can check changed files, only tried accidently on dev branch one dependency because it's recent, i tested and failed, reverted back.

**Review by authorisation — APPROVED** [review]

**Pull request merged**